### PR TITLE
Add mode_num to too_manual

### PR DESCRIPTION
**Pull request by ebellm**
**Does this pull request make any changes to the database?**
No.

Addresses #138

### CODE REVIEW

**Comment by lpsinger** [comment]
Instead of a "mode number" could we have textual descriptions of what the modes are?

**Comment by ebellm** [comment]
I agree that would be a better user experience and less prone to error--unfortunately the complete set of mode numbers is not yet enumerated by the camera team, as the mode number encodes a complex set of configuration parameters that they plan to test on-sky.  I don't think we want to have to redeploy the marshal every time they tweak a parameter value in the readout electronics, so I suggest we stick with the box for now.

**Comment by lpsinger** [comment]
> I don't think we want to have to redeploy the marshal every time they tweak a parameter value in the readout electronics

Why not? I think redeploying the marshal is entirely reasonable and expected when the values of the camera mode enumeration have changed.

**Comment by ebellm** [comment]
The camera team can and I expect will deploy new modes during testing within the night--if we have to redeploy the marshal to reflect those changes it will severely hinder testing velocity.

File: growth/too/templates/plan_manual.html [hunk 1]
```diff
@@ -60,6 +60,14 @@
                     <label for=fieldids>Field IDs</label>
                     {{ render_field(form.field_ids, placeholder='comma or space separated list') }}
                 </div>
+                <div class="form-group col-sm-4">
+                    <label for=mode_num>Mode Number (ZTF Only)</label>
+                    {{ render_field(form.mode_num, placeholder='Mode number') }}
+                <small id="balanceHelpBlock" class="form-text text-muted">
```
Comment (lpsinger) [inline growth/too/templates/plan_manual.html:66]: Mismatched indentation

**Comment by ebellm** [comment]
(I'll add that the team is pushing to test on-sky tonight, so if we are able to get this through today it would be helpful.)

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by lpsinger** [comment]
> The camera team can and I expect will deploy new modes during testing within the night--if we have to redeploy the marshal to reflect those changes it will severely hinder testing velocity.

In that case, how would we even know which mode number to request?

**Comment by ebellm** [comment]
The scientist (@igorandreoni in this most immediate case) will be in close communication with the camera team during the testing period.

**Review by lpsinger — APPROVED** [review]

**Comment by lpsinger** [comment]
The CI pipeline is failing and actually dumping core. I have never seen that before! But it's clearly unrelated to this PR, so I'll merge now. We'll need to fix the CI pipeline and the Docker Hub builds in order to deploy, though.

**Pull request merged**